deepset_cloud_sdk/_api/config.py

```diff
@@ -12,48 +12,81 @@
 
 
 def load_environment() -> bool:
-    """Load environment variables from the .env file.
+    """Load environment variables using a cascading fallback model.
 
-    If an .env file exists in the current directory, load the environment variables from there.
-    Otherwise, load the environment variables from the .env file in the home directory. You can create this file using the CLI
-    by running `deepset-cloud login` in the terminal.
+    1. Load local .env file in current directory if it exists
+    2. For additional variables, load from global ~/.deepset-cloud/.env
+    3. Environment variables can override both
 
-    :return: True if the environment variables were loaded successfully, False otherwise.
+    :return: True if required environment variables were loaded successfully, False otherwise.
     """
-    successfully_loded_env: bool = False
+    successfully_loaded_env: bool = False
     current_path_env = os.path.join(os.getcwd(), ".env")
-    if os.path.isfile(current_path_env):
-        # Load the environment variables from the .env file in the current directory
-        successfully_loded_env = load_dotenv(current_path_env)
-        return successfully_loded_env
+    global_env_exists = os.path.isfile(ENV_FILE_PATH)
+    local_env_exists = os.path.isfile(current_path_env)
+
+    # First load local config
+    if local_env_exists:
+        successfully_loaded_env = load_dotenv(current_path_env)
+        logger.debug(f"Loaded local .env file at {current_path_env}.")
```

Comment:
Given Oliver's feedback it might be worth making this visible by default to the user by changing it to `info` - what do you think? 
```suggestion
        logger.info(f"Loaded local .env file at {current_path_env}.")
```